AudioSinkExample/AudioSinks/ExampleSpeechRecognizer.h

```diff
@@ -0,0 +1,25 @@
+//
+//  ExampleSpeechRecognizer.h
+//  AudioSinkExample
+//
+//  Copyright © 2017 Twilio, Inc. All rights reserved.
+//
+
+@import Foundation;
+@import Speech;
+
+#import <TwilioVideo/TwilioVideo.h>
+
+@interface ExampleSpeechRecognizer : NSObject <TVIAudioSink>
+
+- (null_unspecified instancetype)initWithAudioTrack:(nonnull TVIAudioTrack *)audioTrack
+                                         identifier:(nonnull NSString *)identifier
+                                      resultHandler:(void (^ _Nonnull)(SFSpeechRecognitionResult * __nullable result, NSError * __nullable error))resultHandler;
+
+// Breaks the strong reference from TVIAudioTrack by removing its Sink.
+- (void)stopRecognizing;
+
+@property (nonatomic, copy, readonly, nullable) NSString *speechResult;
```

Comment:
Does this property need to be public? Also, since we have `initWithAudioTrack:identifier:resultHandler` takes a block with result argument, can we not remove it completely from the class?

Reply:
It's not used by `ViewController`, but I think its useful to think of `ExampleSpeechRecognizer` as a self contained object which can be queried  for its latest state. If you feel really strongly I can remove it.

Reply:
Add a comment explaining when this might be used.